ui/src/pages/Feed.svelte

```diff
@@ -119,6 +119,16 @@
       )}`
     );
   };
+
+  const recommendedApps = [
+      { struc: 'app', ship: '~paldev', cord: 'pals', time: '' },
+      { struc: 'app', ship: '~nodmyn-dosrux', cord: 'radio', time: '' }
+  ]
+
+  const events = [
+      { struc: 'app', ship: '~paldev', cord: 'pals', time: '' },
+      { struc: 'app', ship: '~nodmyn-dosrux', cord: 'radio', time: '' }
+  ]
```

Comment:
Unused? Can we remove this?

Reply:
you're viewing an outdated version. this code does not exist ✨